content/platforms.md

```diff
@@ -70,17 +70,17 @@ versions for Chef Infra Client:
 </tr>
 <tr class="even">
 <td>Amazon Linux</td>
-<td><code>x86_64</code></td>
+<td><code>x86_64</code>, <code>aarch64</code> (2.x only)</td>
 <td>2013+ and 2.0</td>
 </tr>
 <tr class="odd">
 <td>CentOS</td>
-<td><code>x86_64</code></td>
```

Comment:
can we try adjusting the column widths for this table? the arch column is all bunched up with a ton of space to the left of it.

Reply:
I've adjusted the columns